std/algorithm/comparison.d

```diff
@@ -116,7 +118,7 @@ if (isExpressionTuple!values)
 }
 
 ///
-@safe unittest
+@safe @betterC unittest
```

Comment:
This won't be displayed on dlang.org yet, but the next step would be to figure out how to get Ddoc / Ddox to do so.
(it will be a lot easier for Ddox though.)